Src/BootCamp.Chapter/Computer/Body.cs

```diff
@@ -6,5 +6,11 @@ namespace BootCamp.Chapter.Computer
 {
     public class Body
     {
+        private string _body;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_body' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
is used

Reply:
is used